diff -Naur a/include/namespace.h b/include/namespace.h
--- a/include/namespace.h	2018-06-04 03:57:02.791445898 -0400
+++ b/include/namespace.h	2020-01-14 00:08:43.326869109 -0500
@@ -31,14 +31,14 @@
 # endif
 
 # if !defined(HAVE_UNSHARE) && defined(SYS_unshare)
-static inline int unshare(int flags)
+inline int unshare(int flags)
 {
 	return syscall(SYS_unshare, flags);
 }
 # endif
 
 # if !defined(HAVE_SETNS) && defined(SYS_setns)
-static inline int setns(int fd, int nstype)
+inline int setns(int fd, int nstype)
 {
 	return syscall(SYS_setns, fd, nstype);
 }
diff -Naur a/sys-utils/prlimit.c b/sys-utils/prlimit.c
--- a/sys-utils/prlimit.c	2019-04-24 05:02:03.125448783 -0400
+++ b/sys-utils/prlimit.c	2020-01-14 00:08:51.246789205 -0500
@@ -143,11 +143,11 @@
 
 #ifndef HAVE_PRLIMIT
 # include <sys/syscall.h>
-static int prlimit(pid_t p, int resource,
-		   const struct rlimit *new_limit,
-		   struct rlimit *old_limit)
+int prlimit(__pid_t __pid, enum __rlimit_resource __resource,
+		   const struct rlimit *__new_limit,
+		   struct rlimit *__old_limit)
 {
-	return syscall(SYS_prlimit64, p, resource, new_limit, old_limit);
+	return syscall(SYS_prlimit64, __pid, __resource, __new_limit, __old_limit);
 }
 #endif
 
